rcldotnet_tests/package.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
```

Comment:
This package should be integrated into `rcldotnet`, see for example https://github.com/ros2/rclpy/blob/master/rclpy/CMakeLists.txt#L146-L168 or https://github.com/esteve/ros2_java/blob/master/rcljava/CMakeLists.txt#L173-L303

Reply:
I'm sorry to tell you that my knowledge of cmake does not allow me to go further without help.

I have been looking at how to integrate xunit + (cmake || ament), and I have not found anything.

I would not like this PR to be blocked because of this. This PR brings much functionality, and there is another PR #11 waiting with even more functionality, could we leave the integration with xunit + ament for later? Using dotnet test is not the desirable or definitive solution, but it allows us to do tests.

Reply:
> I'm sorry to tell you that my knowledge of cmake does not allow me to go further without help.

I've pointed you to a couple of examples of how to integrate tests with CMake. Please ask in detail what parts of the integration you're struggling with and I'll help, but just "asking for help" without any concrete question would be very time-consuming.

> I have been looking at how to integrate xunit + (cmake || ament), and I have not found anything.

CMake does not have builtin support for either Python nor Java, however you can have a look at `rclpy` and `rcljava` and how tests are integrated.

> I would not like this PR to be blocked because of this.

Why not? This is a matter of good engineering practices. And I would like not to merge delicate code that is not tested and that can cause problems in the future. That's why I set up the CI, to ensure that, at least, everything can be built, but given that you're using this library, I assume you'd like to be able to depend on it.

> This PR brings much functionality, and there is another PR #11 waiting with even more functionality, could we leave the integration with xunit + ament for later?

Sorry, but while any functionality is more than welcomed no matter how small it is, asking for it to be merged using that argument is not the best way, honestly I'd appreciate if you phrased it in a more constructive way.

> Using dotnet test is not the desirable or definitive solution, but it allows us to do tests.

Those tests are not run automatically and that's a big problem, because any regression would go unnoticed and cause a lot of time to be wasted if there's an issue in the future.

Reply:
Done!!